### DISCUSSION TITLE
tutorials/voyage-part1/06-publishing-the-subgraphs-with-rover

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#70] [course feedback]
# tutorials/voyage-part1/06-publishing-the-subgraphs-with-rover

Push your subgraph schemas to the Apollo Schema Registry using the Rover command-line interface (CLI)

https://www.apollographql.com/tutorials/voyage-part1/06-publishing-the-subgraphs-with-rover

**Comment by wesleyscholl** [comment]
This should be the `APOLLO_GRAPH_REF`: 
Example - `flyby-1qurtx`
```bash
<name-xxxxxx>
```

**Reply by mabuyo** [reply]
`APOLLO_GRAPH_REF` is a combination of the supergraph ID and the variant, with an `@` symbol in between.

**Comment by gustvoassuncao** [comment]
There's no button anymore on the Graph Section. To be able to follow the lesson I had to go to the Dashboard section and scrolled to the NO. OF GRAPHS Card and I could see the option to Create the Graph. I don't know if it was an update but for me it just worked this way. If someone is facing the same issue, it can help u.

**Comment by ecguerra** [comment]
Is the Enterprise plan still a requirement for this course? I tried going through the free trial links but it wanted me to make a new account. Logging in just takes me back to my regular homepage without the offer. The only option I have under the Enterprise plan is "Talk to Sales" which feels like a waste of everyone's time if it's just for this course.

**Reply by mabuyo** [reply]
Hey there, sorry for the confusion! You should be able to complete the course with the latest free plan.

**Comment by BillLenoir** [comment]
When I go to GraphOS Studio and click Connect Your API, I do not see the modal window you describe. I am on the free plan. The view I do see asks me for my GraphQL endpoint. When I enter http://localhost:4001/, it tells me the end point must be publicly accessible. I do not have a publicly accessible end point. What do I need to do?

**Reply by mabuyo** [reply]
Hi @BillLenoir - it sounds like you might be on an older, legacy free plan. Are you able to check the Organization tab -> Billing and usage to confirm what your plan type says? Or if you could include a screenshot of the modal you're seeing, I can try to troubleshoot from there.

**Reply by BillLenoir** [reply]
Thanks, @mabuyo, for the quick response. My Plan type is Serverless (Free)

Here is a screen shot of what I'm seeing when I try to connect my API: https://s3.us-east-1.amazonaws.com/data.bill-lenoir.com/Screenshot+2025-12-11+at+10.22.20.png

**Reply by BillLenoir** [reply]
Well, actually, that's where the entry point lives. Clicking that button takes me here: https://s3.us-east-1.amazonaws.com/data.bill-lenoir.com/Screenshot+2025-12-11+at+10.38.55.png

**Reply by BillLenoir** [reply]
I am pressed for time, so I upgraded my account to Developer, but that doesn't seem to have helped. This is what I see now: https://s3.us-east-1.amazonaws.com/data.bill-lenoir.com/Screenshot+2025-12-14+at+10.27.01.png

**Reply by mabuyo** [reply]
Hi @BillLenoir - so sorry for the delayed response! If you do come back to this, you'll want to click on "Add existing graph" from that page, which will walk you through the flow in this lesson.

To clarify, the Serverless (Free) was an older legacy plan which is why you weren't seeing the correct modal. We do need to update the screenshots to reflect the new UI though!

**Comment by usmangh** [comment]
Hi , wby do I get emails when you reply to somebody else?
Thanks,Usman.

Yahoo Mail: Search, organise, conquer 
 
  On Wed, 7 Jan 2026 at 7:23 am, Michelle ***@***.***> wrote:   


Hi @BillLenoir - so sorry for the delayed response! If you do come back to this, you'll want to click on "Add existing graph" from that page, which will walk you through the flow in this lesson.

To clarify, the Serverless (Free) was an older legacy plan which is why you weren't seeing the correct modal. We do need to update the screenshots to reflect the new UI though!

**Reply by mabuyo** [reply]
Hi there! That's strange, this might be a GitHub-specific issue? I see the last line says "You are receiving this because you were mentioned", but I don't see your username mentioned anywhere in the discussion 🤔 It's also possible you're subscribed to replies in this post, try unsubscribing. Hope this helps.
